doc/main_window.rst

```diff
@@ -43,6 +43,7 @@ Overview
    7   Play-head / Ruler   The ruler shows the time-scale, and the red line is the play-head. The play-head represents the current playback position.
    8   Timeline            The timeline visualizes your video project, and each clip and transition in your project.
    9   Filter              Filter the list of items shown (project files, transitions, and effects) by using these buttons and filter textbox. Enter a few letters of what you are looking for, and the results will be shown.
+   10  Filter              Right to left: Jump to Start, Rewind, Play/Pause, Fast Forward, and Jump to End Buttons
```

Comment:
"Filter" is... incorrect. "Transport" or "Playback" or something seems roughly accurate.

Also, contrary to the description you actually list them left-to-right (as one would expect).

Reply:
Thanks! copied the line above for the styling. Totally forgot to change the name. Fixed